doc/examples.md

```diff
@@ -263,4 +263,32 @@ Setting the hostname of a system is as simple as writing `/etc/hostname`:
 }
 ```
 
+## Adding users
+
+Users can be added to a Container Linux system with the `passwd.users` key which takes a list of objects that specify a given user. If you wanted to configure a user "systemUser" and a user "jenkins" you would do that as follows:
```

Comment:
Sorry, my last comment wasn't clear. Ignition is meant to be distro-agnostic. There should be no mention of Container Linux in the documentation.